### ISSUE TITLE
visualization evaluation

### ISSUE THREAD

**Issue by Dean-UQ** [LvXinTao/HIMO_dataset#5]
Hi,
  Nice work! Do you plan to release the visualization code to evaluate the synthesized HOI samples qualitatively?
  Many Thanks

**Comment by LvXinTao** [comment]
Yes, we will release that part of code. Before that you can refer to [HOI-Diff](https://github.com/neu-vi/HOI-Diff/tree/main/visualize) for the visualization code.
